addon/components/single-document-details.hbs

```diff
@@ -250,14 +250,18 @@
         <ul class="uk-list uk-list-divider version-history">
           {{#each (filter-by "variant" "original" @document.files) as |file|}}
             <li data-test-file class="uk-flex">
-              <span class="uk-margin-right">
-                {{format-date
+              <span class="uk-margin-right" title={{format-date
                   file.createdAt
                   month="2-digit"
                   day="2-digit"
                   year="numeric"
                   hour="2-digit"
-                  minute="2-digit"
+                  minute="2-digit"}}>
+                {{format-date
+                  file.createdAt
+                  month="2-digit"
+                  day="2-digit"
+                  year="numeric"
```

Comment:
Let's try the two-row style instead, which should also work well on smaller screens